src/ru/skypro/Main.java

```diff
@@ -3,6 +3,92 @@
 public class Main {
 
     public static void main(String[] args) {
-	// write your code here
+
+        //Задание 1
+
+        int clientOS = 1;
+        if (clientOS == 1) {
+            System.out.println("Установите версию приложения для Android по ссылке");
+        }
+        if (clientOS == 0) {
+            System.out.println("Установите версию приложения для IOS по ссылке");
+        }
+
+
+        // Задание 2
+
+        int clientDeviceYear = 2015;
+        int clientOs = 0;
+        if (clientDeviceYear < 2015 && clientOs == 1)
+            System.out.println("Установите облегченную версию приложения для Android по ссылке");
+        if (clientDeviceYear >= 2015 && clientOs == 1) {
+            System.out.println("Установите обычную версию приложения для Android по ссылке");
+        } else if (clientDeviceYear < 2015 && clientOs == 0) {
+            System.out.println("Установите облегченную версию приложения для iOS по ссылке");
+        } else if (clientDeviceYear >= 2015 && clientOs == 0) {
+            System.out.println("Установите обычную версию приложения для iOS по ссылке");
+        }
+
+
+        // Задание 3
+
+        int year = 2021;
+
+        if (year % 4 == 0 && year % 100 != 0 || year % 400 == 0) {
+            System.out.println(year + " год является високосным!");
+        } else {
+            System.out.println(year + " год не является високосный!");
+        }
+
+        //Задание 4
+        int deliveryDistance = 101;
+        int distanceOne = 1;
+
+        if (deliveryDistance < 20) {
+            System.out.println("Потребуется дней:" + distanceOne);
+        }else if (deliveryDistance >= 20 && deliveryDistance < 60) {
+            System.out.println("Потребуется дней:" + distanceOne*2);
+        }else if (deliveryDistance >= 60 && deliveryDistance < 100) {
```

Comment:
левое условие всегда true, поскольку не выполняется условие выше, можно упростить
аналогично и для других условий
```suggestion
        }else if (deliveryDistance < 100) {
```